devel/gdb/Portfile

```diff
@@ -56,13 +55,21 @@ compiler.blacklist      {clang < 601}
 # compiler supports the standard - getting rid of old Apple GCC versions and the like?
 compiler.cxx_standard   2011
 
+configure.dir ${workpath}/build
+configure.cmd ${worksrcpath}/configure
```

Comment:
This isn't already the default configure command?

Reply:
Not after I change `configure.dir`, apparently. See below.